### PR TITLE
Gwh apidocs 1730 aws entity resolution 02

### PR DESCRIPTION
**Pull request by genwhittTTD**
AWS entity resolution doc

### CODE REVIEW

**Comment by sunnywu** [comment]
On top of these changes, should we add the latest AWS ER+UID2 announcement on the unifiedid.com front page (Press releases and media coverage section)?

and add this link?
https://aws.amazon.com/blogs/industries/aws-entity-resolution-expands-data-matching-capabilities-with-liveramp-transunion-and-unified-id-2-0-integrations/

File: docs/guides/integration-aws-entity-resolution.md [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+
+3. Create an IAM role for a console user.
+
+4. Create a workflow job role for AWS Entity Resolution.
+
+5. Create schema mapping for the data: see [Creating a schema mapping](https://docs.aws.amazon.com/entityresolution/latest/userguide/create-schema-mapping.html) in the AWS documentation.
+
+## Create a Matching Workflow
+
+When your initial account creation and setup are done, the next step is to create a matching workflow for your data.
```
Comment (sunnywu) [inline docs/guides/integration-aws-entity-resolution.md:130]: should we mention the instructions below are from The AWS Entity Resolution User Guide https://docs.aws.amazon.com/entityresolution/latest/userguide/create-matching-workflow-provider.html#create-mw-uid

here?

File: docs/guides/integration-aws-entity-resolution.md [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+:::
+
+The following steps are a summary of actions you'll take in the AWS Entity Resolution user interface. For additional details and instructions, consult [the AWS Entity Resolution documentation](https://docs.aws.amazon.com/entityresolution/).
+
+To create the matching workflow, first sign in to the AWS Management console, open the **AWS Entity Resolution** page, and choose **Workflows** > **Matching workflows** > **Create matching workflow**. Then, complete the following steps for your matching workflow.
+
+1. Specify the following details for the matching workflow:
+   - **Name**: This value is required. We recommend a short, clear, intuitive name. For example, with your own company name: `CompanyName_UID2_Daily_Matching`.
+   - **Description**: An optional description for the matching workflow.
+   - **Data input**: The source for your data in AWS. From the drop-down list, select an AWS Glue database, then the AWS Glue table, and then the schema mapping. You can specify more than one data input. The following example shows one data input, email:
```
[the remark this answers is not in the document]
Comment (genwhittTTD) [reply docs/guides/integration-aws-entity-resolution.md:143]: Reworded to: "In the following example, email is the only type of data input:"

File: docs/guides/integration-aws-entity-resolution.md [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+
+### Create UID2 Account
+
+Ask your UID2 contact to register you as a UID2 participant. If you're not sure who to ask, see [Contact Info](../getting-started/gs-account-setup.md#contact-info).
+
+Send your UID2 contact the AWS account ID that you'll use for your UID2 integration on Entity Resolution.
+
+### Subscribe to UID2 on AWS
+
+To participate in UID2 on the Amazon Data Exchange Marketplace, subscribe to the UID2 service. See [Unified ID 2.0 Operator on AWS](https://aws.amazon.com/advertising-marketing/solutions/uid2/).
```
[the remark this answers is not in the document]
Comment (genwhittTTD) [reply docs/guides/integration-aws-entity-resolution.md:99]: @asloobq please let me know on this.

File: docs/guides/integration-aws-entity-resolution.md (hunk 1 above)
Comment (genwhittTTD) [reply docs/guides/integration-aws-entity-resolution.md:130]: @sunnywu we already had a link + reference to the AWS doc, but a little further down. Your link was better. I used yours and moved it up. See update. Thx much.

File: docs/guides/integration-aws-entity-resolution.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (asloobq) [reply docs/guides/integration-aws-entity-resolution.md:99]: I agree with @sunnywu. Here is a screenshot of our private listing
amazon data exchange uid2 private offer screenshot

**Review by asloobq — COMMENTED** [review]
LGTM. Can be approved with addressing sunny's comment on data exchange

File: docs/guides/integration-aws-entity-resolution.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (genwhittTTD) [reply docs/guides/integration-aws-entity-resolution.md:99]: Done, see update. Used the other screen cap.

File: docs/guides/integration-aws-entity-resolution.md [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+
+On receipt of your subscription request, the UID2 team do the following:
+- Review your subscription request.
+- Match your request with the AWS account ID that you sent in [Create UID2 Account](#create-uid2-account).
+- Approve the request.
+
+If you send your request via your UID2 contact, you'll receive a custom invitation to the UID2 private service listing on AWS, as shown in the following example.
```
Comment (asloobq) [inline docs/guides/integration-aws-entity-resolution.md:106]: Its important to convey that this is a temporary arrangement  and the only way until we make the listing public on ADX. May be add more detail like this
> Currently, you need to send your request via your UID2 contact, then you'll receive a custom invitation to the UID2 private service listing on AWS, as shown in the following example, that you need to accept.

Comment (genwhittTTD) [reply docs/guides/integration-aws-entity-resolution.md:106]: Thx. Done, almost identical wording: "Currently, you need to send your request via your UID2 contact. Then, you'll receive a custom invitation to the UID2 private service listing on AWS, as shown in the following example, and must accept the invitation to continue."

File: docs/guides/integration-aws-entity-resolution.md [hunk 5]
```diff
@@ -0,0 +1,197 @@
...
+
+Ask your UID2 contact to register you as a UID2 participant. If you're not sure who to ask, see [Contact Info](../getting-started/gs-account-setup.md#contact-info).
+
+Send your UID2 contact the AWS account ID that you'll use for your UID2 integration on Entity Resolution.
+
+### Subscribe to UID2 on AWS
+
+To participate in UID2 on the Amazon Data Exchange Marketplace, subscribe to the UID2 service. See [Unified ID 2.0 Operator on AWS](https://aws.amazon.com/advertising-marketing/solutions/uid2/).
+
+On receipt of your subscription request, the UID2 team do the following:
```
[the remark this answers is not in the document]
Comment (genwhittTTD) [reply docs/guides/integration-aws-entity-resolution.md:105]: Fixed.  You got me. Thank you!

File: docs/guides/integration-aws-entity-resolution.md [hunk 6]
```diff
@@ -0,0 +1,197 @@
...
+
+For details, see [Getting started: Are you a first-time AWS user?](https://docs.aws.amazon.com/accounts/latest/reference/welcome-first-time-user.html) in the AWS documentation.
+
+### Create UID2 Account
+
+Ask your UID2 contact to register you as a UID2 participant. If you're not sure who to ask, see [Contact Info](../getting-started/gs-account-setup.md#contact-info).
+
+Send your UID2 contact the AWS account ID that you'll use for your UID2 integration on Entity Resolution.
+
+### Subscribe to UID2 on AWS
```
[the remark this answers is not in the document]
Comment (genwhittTTD) [reply docs/guides/integration-aws-entity-resolution.md:97]: Done. Thx.

File: docs/guides/integration-aws-entity-resolution.md [hunk 7]
```diff
@@ -160,7 +160,7 @@ To create the matching workflow, first sign in to the AWS Management console, op
    - Then, under **Partner services**, choose **Unified ID2.0**.
    - Click **Next**.
 
-     If you've subscribed to the Unified ID 2.0 product on Amazon Data Exchange (see [Subscribe to UID2 on AWS](#subscribe-to-uid2-on-aws)), you'll see an **Approved** message.
+     If you've subscribed to the Unified ID 2.0 product on Amazon Data Exchange (see [Subscribe to UID2 on AWS Data Exchange](#subscribe-to-uid2-on-aws-data-exchange)), you'll see an **Approved** message.
```
Comment (asloobq) [inline docs/guides/integration-aws-entity-resolution.md:163]: 'AWS Data Exchange' for consistency

**Review by asloobq — COMMENTED** [review]
small nitpicking comments. Otherwise LGTM

**Review by asloobq — APPROVED** [review]

**Pull request merged**